.github/workflows/CD.yaml

```diff
@@ -30,3 +30,5 @@ jobs:
         run: ./deployment/deploy_to_preview.sh
         env:
           SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      - name: check deployment
```

Comment:
In der CI laden wir die Artefacts von cypress hoch (den cypress order, siehe testing/execute_e2e_tests.sh line 7 für Pfad), die einem ein einfaches debuggen erlauben da dort die videos und Bilder der Tests drin sind. Dies könnte man durch einen 4 Zeilen hier auch machen. Würde ich persönlich hilfreich finden.

Reply:
![image](https://user-images.githubusercontent.com/47553592/163330732-2b67bfe6-ec97-4f3d-bbe0-b0f502145fba.png)
done